2016/cities.html

```diff
@@ -0,0 +1,147 @@
+      {% include tweet.html user_display_name="Plexus Design" user_handle="plexusdesigner" tweet_html='<a href="https://fixmystreet.com">fixmystreet.com</a> 3 min form has fixed an issue that has been bugging me and my neighbours for months! Thank you! <a href="https://twitter.com/mysociety">@mysociety</a>' tweet_url="https://twitter.com/plexusdesigner/status/709519365603991552" date="14 Mar 2016"  %}
+    </div>
+  </div>
+</div>
+
+<div class="section section--white section--cities-councils">
+  <h1>Huge improvements on FixMyStreet for Councils…</h1>
+  <p>FixMyStreet has been totally transformed for some of our most important users: the council staff who receive reports. It’s all thanks to our collaboration with some of the UK’s most forward-thinking councils.</p>
+  <p>Working with <strong>Oxfordshire County Council</strong>, we’ve <a href="https://www.google.com/url?q=https://www.fixmystreet.com/about/council&sa=D&ust=1481717542577000&usg=AFQjCNFo7BqUVyBlHNA9ljbGruZZXcJzag">dramatically improved the feature set of FixMyStreet for Councils</a>, allowing council workers to track and deal with reports more efficiently.</p>
+  <p>And <strong>Bristol Council</strong> have led the way in showing how to replace outdated and expensive suppliers, while still retaining future flexibility, <a href="https://www.mysociety.org/2016/11/22/something-in-the-middle-how-bristol-connects/">through the power of middleware</a>.</p>
+</div>
+
+<div class="section section--white section--flipped">
+  <h1>…And for our community around the world</h1>
+  <p>Since FixMyStreet is an open source project, many of the improvements we’ve made for our UK councils—like <a href="http://fixmystreet.org/2016/11/29/v2.0-html-emails/">HTML emails</a> and <a href="http://fixmystreet.org/2015/10/23/v1.7/">easy report filtering</a>—will also benefit our international re-users. It’s win-win!</p>
+</div>
+
+<div class="section section--no-padding">
+  <img src="{{ site.baseurl }}/assets/img/kota-damansara-blue.jpg" alt="" style="width: 100%;">
+</div>
+
+<div class="section section--white">
+  <h1>FixMyStreet fights corruption in Malaysia</h1>
+  <p>We know, we were surprised too! <a href="https://www.mysociety.org/2016/10/07/when-funds-go-missing-how-fixmystreet-is-helping-fight-corruption-in-malaysia/">Here’s how it happened.</a></p>
+</div>
+
+<div class="section section--white">
+  <h1>MapIt helps people know their rights and campaign for better</h1>
+  <p>This year we heard about some very worthwhile uses of <a href="https://mapit.mysociety.org/">MapIt</a>:</p>
+
+  <ul>
+    <li><a href="https://www.mysociety.org/2016/01/22/how-socialcareinfo-matches-people-to-resources/">Matching people to Social Care resources</a></li>
+    <li><a href="https://www.mysociety.org/2016/08/01/lasas-simplifying-universal-credit-with-mapit/">Simplifying Universal Credit</a></li>
+    <li><a href="https://socialcare.leonardcheshire.org">Assessing cuts in community care</a></li>
+  </ul>
+</div>
+
```

Comment:
Add:
<H2>What can MapIt do for you?</H2>
If you need to match postcodes to constituencies or any other type of point to boundaries, take a look at our <a href="http://mapit.mysociety.org/licensing">very reasonable terms</a>. MapIt's API make integration into your products or services a breeze, and our volume pricing structure means it's within reach whatever the size of your project.